operations/companies.md

```diff
@@ -236,12 +242,22 @@ Adds a new company to the enterprise.
 | `Contact` | string | optional | Other contact details, such as telephone, email or similar. |
 | `Notes` | string | optional | Notes of the company. |
 | `Iata` | string | optional | Iata of the company. |
+| `Department` | string | optional | The internal segmentation of a company, f.e. sales department. |
+| `DunsNumber` | string | optional | The Dun & Bradstreet unique 9-digit DUNS number. |
+| `CreditRating` | [Credit rating parameters](#credit-rating-parameters) | optional | Basic credit rating to define creditworthiness of the company. |
 
 #### Company options parameters
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Invoiceable` | bool | required | Whether the company is invoiceable or not. |
+| `Invoiceable` | boolean | required | Whether the company is invoiceable or not. |
+| `AddFeesToInvoices` | boolean | required | Whether the company has an additional fee applied for invoicing or not. |
+
+#### Credit rating parameters
```

Comment:
I notice this data object appears three times in the page under different names, one for `Get all companies` (called Credit rating), one for `Add company` (called Credit rating parameters) and one for `Update company` (called Credit rating update value). I think we should do one of two things: either (i) just define it once and all operations can reference this one definition; or (ii) if there is a good reason to define it three times (e.g. they are expected to change in the near future), then fine but can we use some sort of naming convention, e.g. Credit rating Get, Credit rating Add, Credit rating Update? Thanks

Reply:
I unified for getAll and add and referenced the same one but for update value I left it as the implementation is different as we use UpdateValues there vs the primitive types in getAll and add operations.

Reply:
![image](https://user-images.githubusercontent.com/22341887/195598181-bcf3a6aa-5a57-42c0-b964-cfab59e085a0.png)

Reply:
@MikeAdamsMews could you please have a look? thx